### PR TITLE
clean ecephys tutorial

### PR DESCRIPTION
**Pull request by bendichter**
## Motivation

some minor adjustments

### CODE REVIEW

File: docs/gallery/domain/ecephys.py [hunk 1]
```diff
@@ -183,27 +181,19 @@
...
 
 ####################
-# NWB organizes data into different groups depending on the type of data. Groups can be thought of
-# as folders within the file. Here are some of the groups within an :py:class:`~pynwb.file.NWBFile` and the types of
-# data they are intended to store:
-#
-# * **acquisition**: raw, acquired data that should never change
-# * **processing**: processed data, typically the results of preprocessing algorithms and could change
-# * **analysis**: results of data analysis
-# * **stimuli**: stimuli used in the experiment (e.g., images, videos, light pulses)
```
Comment (bendichter) [inline docs/gallery/domain/ecephys.py:193]: moved to basics

**Review by oruebel — DISMISSED** [review]

File: docs/gallery/domain/ecephys.py [hunk 2]
```diff
@@ -73,10 +71,9 @@
 #     :alt: electrodes table UML diagram
 #     :align: center
 #
-# Before creating an :py:class:`~pynwb.ecephys.ElectrodeGroup`, you need to provide some information about the
-# device that was used to record from the electrode. This is done by creating a :py:class:`~pynwb.device.Device`
-# object using the instance method :py:meth:`~pynwb.file.NWBFile.create_device`.
-
+# The electrodes table references a required :py:class:`~pynwb.ecephys.ElectrodeGroup`, which is used to represent a
+# group of electrodes. Before creating an :py:class:`~pynwb.ecephys.ElectrodeGroup`, you must define a
+# :py:class:`~pynwb.device.Device` object using the method :py:meth:`.NWBFile.create_device`.
```
Comment (rly) [inline docs/gallery/domain/ecephys.py:76]:
```suggestion
# :py:class:`~pynwb.device.Device` object using the method :py:meth:`~pynwb.file.NWBFile.create_device`.
```

File: docs/gallery/domain/ecephys.py [hunk 3]
```diff
@@ -73,10 +71,9 @@
 #     :alt: electrodes table UML diagram
 #     :align: center
 #
-# Before creating an :py:class:`~pynwb.ecephys.ElectrodeGroup`, you need to provide some information about the
-# device that was used to record from the electrode. This is done by creating a :py:class:`~pynwb.device.Device`
-# object using the instance method :py:meth:`~pynwb.file.NWBFile.create_device`.
-
+# The electrodes table references a required :py:class:`~pynwb.ecephys.ElectrodeGroup`, which is used to represent a
+# group of electrodes. Before creating an :py:class:`~pynwb.ecephys.ElectrodeGroup`, you must define a
+# :py:class:`~pynwb.device.Device` object using the method :py:meth:`~pynwb.file.NWBFile.create_device`.
```
Comment (rly) [inline docs/gallery/domain/ecephys.py]:
```suggestion
# :py:class:`~pynwb.device.Device` object using the method :py:meth:`.NWBFile.create_device`.
```

File: docs/gallery/domain/ecephys.py (hunk 2 above)
Comment (rly) [inline docs/gallery/domain/ecephys.py:76]: Why does this work? :py:meth:&#96;.NWBFile.create_device&#96;

File: docs/gallery/domain/ecephys.py [hunk 4]
```diff
@@ -355,13 +344,12 @@
...
 # Accessing your data
-# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# ^^^^^^^^^^^^^^^^^^^
 #
-# Data arrays are read passively from the file.
-# Calling the data attribute on a :py:class:`~pynwb.base.pynwb.TimeSeries`
-# such as a :py:class:`~pynwb.ecephys.ElectricalSeries` does not read the data
-# values, but presents an :py:class:`~h5py` object that can be indexed to read data.
-# You can use the ``[:]`` operator to read the entire data array into memory.
+# Data arrays are read passively from the file. Calling the data attribute on a :py:class:`~pynwb.base.pynwb.TimeSeries`
```
Comment (rly) [inline docs/gallery/domain/ecephys.py]:
```suggestion
# Data arrays are read passively from the file. Calling the data attribute on a :py:class:`~pynwb.base.TimeSeries`
```

**Review by rly — APPROVED** [review]

File: docs/gallery/domain/ecephys.py (hunk 2 above)
Comment (bendichter) [reply docs/gallery/domain/ecephys.py:76]: I don't know, but this is the way I have figured out to display method links in the way that I want

**Pull request merged**